src/main/java/seedu/address/logic/commands/ViewContactsCommand.java

```diff
@@ -13,7 +13,7 @@
 
     public static final String COMMAND_WORD = "contacts";
 
-    public static final String MESSAGE_SUCCESS = "Viewing all Contacts.";
+    public static final String MESSAGE_SUCCESS = "Viewing All Contacts.";
```

Comment:
It seems that other commands' MESSAGE_SUCCESS do not capitalize all words

Reply:
This is consistent with the other view commands, changed because it looks better next to those. If we want to revert maybe someone can take another look and change it for all.

Reply:
Ohh okay sure! Then I think can leave it like this!